### PR TITLE
elastic: update to 7.17.7

### PR DESCRIPTION
**Pull request by hydrapolic**
Changes

1) kibana

- use only compatible nodejs version

```bash
--- kibana-bin-7.17.5.ebuild    2022-10-22 03:15:01.695660377 +0000
+++ kibana-bin-7.17.7.ebuild    2022-11-03 07:43:50.797946506 +0000
@@ -24,6 +24,7 @@
        dev-libs/nspr
        dev-libs/nss
        >=net-libs/nodejs-16.13.0
+       <net-libs/nodejs-18
        sys-libs/glibc
 "
```
 
2) Elasticsearch

- Dodir of the empty dir now fails, with keepdir ES fails to start either, we add it to the init script instead.

```bash
--- elasticsearch-7.17.5.ebuild 2022-10-22 03:14:57.767644625 +0000
+++ elasticsearch-7.17.7.ebuild 2022-11-03 08:21:04.269867409 +0000
@@ -50,13 +50,12 @@
        keepdir /var/{lib,log}/${PN}
        fowners ${PN}:${PN} /var/{lib,log}/${PN}
        fperms 0750 /var/{lib,log}/${PN}
-       dodir /usr/share/${PN}/plugins
 
        insinto /etc/sysctl.d
        newins "${FILESDIR}/${PN}.sysctl.d" ${PN}.conf
 
        newconfd "${FILESDIR}/${PN}.conf.4" ${PN}
-       newinitd "${FILESDIR}/${PN}.init.8" ${PN}
+       newinitd "${FILESDIR}/${PN}.init.9" ${PN}
 
        systemd_install_serviced "${FILESDIR}/${PN}.service.conf"
```

```bash
--- elasticsearch.init.8        2022-11-03 08:20:20.010670810 +0000
+++ elasticsearch.init.9        2022-11-03 08:21:04.269867409 +0000
@@ -58,6 +58,7 @@
        checkpath -d -o "${ES_USER}:${ES_GROUP}" -m750 "/var/lib/elasticsearch"
        checkpath -d -o "${ES_USER}:${ES_GROUP}" -m750 "/var/log/elasticsearch"
        checkpath -d -o "${ES_USER}:${ES_GROUP}" -m750 "/run/elasticsearch"
+       checkpath -d -o "${ES_USER}:${ES_GROUP}" -m750 "${ES_HOME}/plugins"
        checkpath -d -o "${ES_USER}:${ES_GROUP}" -m750 "${ES_BASE_PATH}"
        checkpath -d -o "${ES_USER}:${ES_GROUP}" -m750 "${LOG_DIR}"
```

### CODE REVIEW

File: www-apps/kibana-bin/kibana-bin-7.17.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+KEYWORDS="-* ~amd64"
+
+RDEPEND="
+	acct-group/kibana
+	acct-user/kibana
+	dev-libs/expat
+	dev-libs/nspr
+	dev-libs/nss
+	>=net-libs/nodejs-16.13.0
+	<net-libs/nodejs-18
```
Comment (juippis) [inline www-apps/kibana-bin/kibana-bin-7.17.7.ebuild:27]: This is really unfortunate :(

**Review by juippis — APPROVED** [review]
Thanks!

File: www-apps/kibana-bin/kibana-bin-7.17.7.ebuild (hunk 1 above)
Comment (hydrapolic) [reply www-apps/kibana-bin/kibana-bin-7.17.7.ebuild:27]: Agreed :(

**Comment by hydrapolic** [comment]
Thanks @juippis

**Pull request closed without merging**